.dockerignore

```diff
@@ -0,0 +1,3 @@
+/examples/
+/*.*
+/LICENSE
```

Comment:
Should also exclude the `/cwls` directory, only need the dockerfile and any scripts used to physically build the image

Reply:
Added /cwls in https://github.com/cancerit/dockstore-cgpbigwig/pull/1/commits/18bff7fb4275d19d0e5d8ec12e5a98cbae4e3cda